pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -76,6 +76,12 @@ outputs:
       types:
         - alert:
             tagged-packets: yes
```

Comment:
does that determine the generation of `packet` events for rejected connections when logging is enabled?

Reply:
Yes, it determines `event_type: packet` log for rejected connections if `enableLogging: true`.